### ISSUE TITLE
Problem building Nijmegen.QArith contribution

### ISSUE THREAD

**Issue by Zimmi48** [Zimmi48/bugzilla-test#5689]
Note: the issue was created automatically with bugzilla2github tool

# Bugzilla Bug ID: 975
Date: 2005-06-03 17:21:12 +0200
From: @_roconnor
To:   

Last updated: 2005-06-03 17:21:12 +0200

## Bugzilla Comment ID: 1090
Date: 2005-06-03 17:21:12 +0200
From: @_roconnor

Full_Name: Russell O'Connor
Version: 8.0pl2
OS: linux
Submission from: n032202.cs.kun.nl (131.174.32.202)

There are some problems with building the Nijmegen.QArith contributions.

There are two make files: makefile, and Makefile.  makefile is outdated and
should be removed from the contribution.  However Makefile is broken.  It gets
its list of files from Make, and Make has Zaux.v listed with an extra * at the
end.  This breaks Makefile and causes it to go into an infinite loop when
building or cleaning.  The extra * probably stems from the fact that Zaux.v is
executable for no good reason.  This should be fixed.

So in summary, I recommend doing the following:

1. Delete makefile
2. make Zaux.v non-executable.
3. rebuild Make
4. rebuild Makefile

## Bugzilla Comment ID: 1092
Date: 2005-06-03 19:32:20 +0200
From: @_roconnor

Hugo Herbelin <coq-bugs@pauillac.inria.fr>
  Hi,

> So in summary, I recommend doing the following:
> 
> 1. Delete makefile
> 2. make Zaux.v non-executable.
> 3. rebuild Make
> 4. rebuild Makefile

  Thanks very much for this detailed bug report. It is now fixed.

  Hugo Herbelin

## Bugzilla Comment ID: 1091
Date: 2007-07-04 17:03:56 +0200
From: @_roconnor

Fixed in (V8) contrib cvs and on web site
